File: CS/DxSample/Editors/GroupControl.cs
using DevExpress.LookAndFeel;
using DevExpress.Utils.Drawing;

namespace DxSample.Editors {
    public class GroupControl : DevExpress.XtraEditors.GroupControl {
        protected override ObjectPainter CreatePainter() {
            if (this.LookAndFeel != null && this.LookAndFeel.ActiveStyle == ActiveLookAndFeelStyle.Skin)
                return new SimpleGroupObjectPainter(this, this.LookAndFeel.ActiveLookAndFeel);
            return base.CreatePainter();
        }
    }
}

File: VB/DxSample/Editors/GroupControl.vb
Imports DevExpress.LookAndFeel
Imports DevExpress.Utils.Drawing

Namespace DxSample.Editors
	Public Class GroupControl
		Inherits DevExpress.XtraEditors.GroupControl

		Protected Overrides Function CreatePainter() As ObjectPainter
			If Me.LookAndFeel IsNot Nothing AndAlso Me.LookAndFeel.ActiveStyle = ActiveLookAndFeelStyle.Skin Then
				Return New SimpleGroupObjectPainter(Me, Me.LookAndFeel.ActiveLookAndFeel)
			End If
			Return MyBase.CreatePainter()
		End Function
	End Class
End Namespace
